src/mixins/filing-template-mixin.ts

```diff
@@ -693,164 +709,160 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       }) || []
     )
 
-    // Store provisions removed
+    // store Provisions Removed
     this.setProvisionsRemoved(filing.alteration.provisionsRemoved)
 
-    // Store office addresses **from snapshot** (because we don't change office addresses in an alteration)
+    // store Office Addresses **from snapshot** (because we don't change office addresses in an alteration)
     this.setOfficeAddresses(entitySnapshot.addresses)
 
-    // Store people and roles **from snapshot** (because we don't change people and roles in an alteration)
+    // store People And Roles **from snapshot** (because we don't change people and roles in an alteration)
     this.setPeopleAndRoles(entitySnapshot.orgPersons)
 
-    // Store business contact info
+    // store Business Contact
     this.setBusinessContact(entitySnapshot.authInfo.contact)
 
-    // Store share classes and resolution dates
+    // store Share Classes and Resolution Dates
     this.setShareClasses(
       filing.alteration.shareStructure?.shareClasses ||
       entitySnapshot.shareStructure?.shareClasses
     )
     this.setResolutionDates(filing.alteration.shareStructure?.resolutionDates || [])
     this.setOriginalResolutionDates(entitySnapshot.resolutions)
 
-    // Store certify state
+    // store Certify State
     this.setCertifyState({
       valid: false,
       certifiedBy: filing.header.certifiedBy
     })
 
-    // Store Folio Number
+    // store Folio Number
     this.setFolioNumber(entitySnapshot.authInfo.folioNumber || '')
 
-    // If Transactional Folio Number was saved then store it.
+    // if Transactional Folio Number was saved then store it
     if (filing.header.isTransactionalFolioNumber) {
       this.setTransactionalFolioNumber(filing.header.folioNumber)
     }
 
-    // Store document optional email
+    // store Document Optional Email
     this.setDocumentOptionalEmail(filing.header.documentOptionalEmail)
 
-    // Store effective date
+    // store Effective Date
     this.setEffectiveDateTimeString(filing.header.effectiveDate)
     this.setIsFutureEffective(filing.header.isFutureEffective)
 
-    // Store Court Order date
+    // store File Number and POA
     this.setFileNumber(filing.alteration.courtOrder?.fileNumber)
     this.setHasPlanOfArrangement(filing.alteration.courtOrder?.hasPlanOfArrangement)
 
-    // Store Staff Payment data
+    // store Staff Payment
     this.storeStaffPayment(filing)
   }
 
   /**
-   * Parses a draft change of registration filing into the store.
+   * Parses a draft Change of Registration filing into the store.
    * @param filing the change filing
    * @param entitySnapshot the latest entity snapshot
    */
   parseChangeRegFiling (filing: ChgRegistrationFilingIF, entitySnapshot: EntitySnapshotIF): void {
-    // Store business snapshot
-    this.setEntitySnapshot(cloneDeep(entitySnapshot))
```

Comment:
We should not need to clone the snapshot, because I _think_ the store will make a copy of it.

Cloning is useful to prevent local changes from changing the original object.